metabox/metabox/core/lxd_provider.py

```diff
@@ -232,6 +254,13 @@ def _run_transfer_commands(self, machine):
                     "sudo mkdir -p {}".format(os.path.dirname(dest)),
                     verbose=self._debug_machine_setup
                 )
+                # Ensure that the location is not already there
+                #  else cp will copy inside it
+                run_or_raise(
+                    machine._container,
+                    "sudo rm -rf {} 2>/dev/null || true".format(dest),
```

Comment:
The rationale here is: if the directory is there remove it, if it is not, we do not care. The reason this was done is to avoid copying the files inside it as the command following this would do. I do not know a better way to do such a thing with `cp` alone. I'm piping away the error to "hide" the location does not exist error because I would pollute the log with no benefit